docs/getting-started/part-3/metadata.md

```diff
@@ -0,0 +1,87 @@
+---
+id: metadata
+title: Metadata
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Goal
+
+In this part, we are going to dive into the role of descriptive metadata in Smart Data Lake Builder (SDLB) and demonstrate how to define this additional metadata for SDLB objects using the configuration file [solution](../config-examples/application-part3-download-incremental-mode.conf) presented in part 3.
+
+## About Metadata 
+<!--Should we also write about the markdown description files?-->
+Descriptive metadata, in the context of SDLB, refers to additional configuration attributes and markdown files that goes beyond the basic "id" of DataObjects and Actions. It is mainly about adding descriptions and tags to configuration object, and other attributes that help manage and explore data effectively. 
+
+## Metadata and Descriptions
+<!--There is also a metadata.name, where one can define a user friendly name for the configuration object. -->
+
+While metadata configuration attributes are optional, they can be very useful in many situations. `metadata.name` can define a user frinedly and name stringforward for the configuration object. 
```

Comment:
Not sure what is meant with "can define a user friendly and name stringforward for the configuration object"

Do you mean "can define a user friendly and straight forward name for the configuration object" ?

Reply:
yes i will change it